src/Sylius/Bundle/OrderBundle/Resources/config/doctrine/model/Adjustment.orm.xml

```diff
@@ -45,6 +45,10 @@
         <many-to-one field="orderItem" target-entity="Sylius\Component\Order\Model\OrderItemInterface" inversed-by="adjustments">
             <join-column name="order_item_id" referenced-column-name="id" nullable="true" on-delete="CASCADE"/>
         </many-to-one>
```

Comment:
Is the plan to keep adjustments at the orderItem level? Or is this a later phase?